lib/pos_blockchain.go

```diff
@@ -1708,25 +1706,14 @@ func (bc *Blockchain) commitBlockPoS(blockHash *BlockHash, verifySignatures bool
 		// Can't commit a block that's already committed.
 		return errors.Errorf("commitBlockPoS: Block %v is already committed", blockHash.String())
 	}
-	block, err := GetBlock(blockHash, bc.db, bc.snapshot)
-	if err != nil {
-		return errors.Wrapf(err, "commitBlockPoS: Problem getting block from db %v", blockHash.String())
-	}
 	// Connect a view up to the parent of the block we are committing.
-	utxoView, err := bc.getUtxoViewAtBlockHash(*block.Header.PrevBlockHash)
+	utxoViewAndUtxoOps, err := bc.getUtxoViewAtBlockHash(*blockHash)
 	if err != nil {
 		return errors.Wrapf(err, "commitBlockPoS: Problem initializing UtxoView: ")
 	}
-	txHashes := collections.Transform(block.Txns, func(txn *MsgDeSoTxn) *BlockHash {
-		return txn.Hash()
-	})
-	// Connect the block to the view!
-	utxoOpsForBlock, err := utxoView.ConnectBlock(
-		block, txHashes, verifySignatures, bc.eventManager, block.Header.Height)
-	if err != nil {
-		// TODO: rule error handling? mark blocks invalid?
-		return errors.Wrapf(err, "commitBlockPoS: Problem connecting block to view: ")
-	}
+	utxoView := utxoViewAndUtxoOps.UtxoView
+	utxoOpsForBlock := utxoViewAndUtxoOps.UtxoOps
```

Comment:
nit: may as well keep the naming consistent
```suggestion
	utxoOps := utxoViewAndUtxoOps.UtxoOps
```